templates/named.conf.erb

```diff
@@ -24,5 +24,12 @@ acl "<%= acl_name %>"  {
 };
 <%- end -%>
 
+<%- if scope.lookupvar('::dns::additional_directives').any? -%>
+// additional directives
+<% scope.lookupvar('::dns::additional_directives').each do |directive| -%>
+<%= directive %>
+<%- end -%>
+<%- end -%>
```

Comment:
The test filters out empty lines which is a bit of a simplification for us developers, but users might notice it.